### PR TITLE
[docs] [Drivers and ORMs] Update activerecord ORM.

### PR DESCRIPTION
**Pull request by HarshDaryani896**
Made some changes required due to version update of Active Record.
Build preview: https://deploy-preview-14511--infallible-bardeen-164bc9.netlify.app/preview/drivers-orms/ruby/

### CODE REVIEW

**Comment by aishwarya24** [comment]
@HarshDaryani896 , Thanks for the updates. I have pushed more edits to refine the content on the index page for Ruby and also _Learn more_ on each of the pages.

File: docs/content/preview/drivers-orms/ruby/activerecord.md [hunk 1]
```diff
@@ -11,35 +11,67 @@ menu:
...
-```sh
-$ git clone https://github.com/YugabyteDB-Samples/orm-examples.git
-```
+YugabyteDB YSQL API has full compatibility with Active Record ORM for data persistence in Ruby applications.
 
-```sh
-bundle install
-```
+## CRUD operations
+
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/ruby/activerecord.md:19]: any prerequisites for this? active record?

File: docs/content/preview/drivers-orms/ruby/activerecord.md [hunk 2]
```diff
@@ -96,8 +128,14 @@ Type "help" for help.
 yugabyte=#
 ```
 
-```plpgsql
-yugabyte=# SELECT count(*) FROM users;
+Connect to the database that is mentioned in `config/database.yml` file. Default is `ysql_active_record`.
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/ruby/activerecord.md:131]: there is a file named in database.yml? why can't we say what it is? is it the same as the default, or does the default refer to the file in database.yml? if so, who would have changed it?

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/drivers-orms/ruby/activerecord.md (hunk 2 above)
Comment (aishwarya24) [reply docs/content/preview/drivers-orms/ruby/activerecord.md:131]: We mention about what it is on line 30 and that it can be customized by the user.

**Pull request merged**